metadata/com.sparklinlabs.Superpowers.appdata.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>com.sparklinlabs.Superpowers.desktop</id>
```

Comment:
you are missing a `<name>` field. See https://phabricator.endlessm.com/T15178#326355